app/.idea/workspace.xml

```diff
@@ -0,0 +1,397 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be here, why is there an `.idea` folder underneath `app` directory?